src/main/java/com/example/demo/Controllers/MemberController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.Controllers;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import com.example.demo.Models.Member;
+import com.example.demo.Repositories.MemberRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MemberController {
+    private final MemberRepository memberRepository;
```

Comment:
Which Dependency Injection type are you using and why?

Reply:
In this case I'm using the constructor type dependency injection but the @Autowired annotation is missing

Reply:
That is right, since Spring Framework version 4.2. @Autowired annotation is optional, so if you are using a later version of Spring that is why all is working fine.